Makefile

```diff
@@ -130,6 +133,10 @@ ifneq ($(detected_OS),Windows)
     # Fix for failures due to 'can't allocate code signature data for'
     NIM_PARAMS += --passL:"-headerpad_max_install_names"
     NIM_PARAMS += --passL:"-F$(QT5_LIBDIR)"
+
+    HUNSPELL := bottles/hunspell/lib/libhunspell-1.7.a
```

Comment:
So we're linking statically on macOS, but on Linux and Windows we're relying on `linuxdeployqt` / `windeployqt` to copy the shared library into the AppImage (Linux) or `bin/` containing the DLLs (Windows). Is that correct?

On Linux, I guess hunspell would be installed with `apt`, etc. But what about on Windows?

Reply:
For linux yes, but for Windows i dont know... i see than we should build it from sources

Reply:
I just checked and it's available as an MSYS2 package: https://packages.msys2.org/base/mingw-w64-hunspell.

But to use it the Windows build logic (Makefile and Jenkins stuff) would need some refactors to assume the development/build context is an MSYS2 environment.

Hunspell is available in someone's [personal scoop bucket](https://github.com/iquiw/scoop-bucket), but it also involves that person's build of hunspell as opposed to an official build/download, so we probably don't want to use that.